src/Makefile

```diff
@@ -467,7 +467,7 @@ INCLUDE_PATHS = -I.
 ifeq ($(TARGET_PLATFORM),PLATFORM_DESKTOP_GLFW)
     INCLUDE_PATHS += -Iexternal/glfw/include
     ifeq ($(PLATFORM_OS),BSD)
-        INCLUDE_PATHS += -I/usr/local/include
+        INCLUDE_PATHS += -I/usr/local/include -I/usr/pkg/include -I/usr/X11R7/include
```

Comment:
Those paths look highly irregular, why are they needed? Is there no other option?

Reply:
because netbsd installs packages under /usr/pkg